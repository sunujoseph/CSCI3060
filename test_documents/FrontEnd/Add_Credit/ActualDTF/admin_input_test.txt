06 userBS          AA 123556.78
00 admin1          AA 000150.78
