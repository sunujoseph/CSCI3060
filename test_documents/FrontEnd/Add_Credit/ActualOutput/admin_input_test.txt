path recieved: C:\Users\pwner\Documents\Uoit\Year 5 Semester 2\CSCI 3060U\Project\CSCI3060\test_documents\FrontEnd\Add_Credit\ActualDTF\admin_input_test.txt
Enter Username: temp: |
addcreditÍÍÍÍÍÍÍıııı|
username: |admin1
addcredi| not found
Error: Not Logged In
Error: Not Logged In
Error: Not Logged In
Error: Not Logged In
