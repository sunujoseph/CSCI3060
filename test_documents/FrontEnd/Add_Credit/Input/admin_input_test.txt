login
admin1
addcredit
userBS
100
logout
