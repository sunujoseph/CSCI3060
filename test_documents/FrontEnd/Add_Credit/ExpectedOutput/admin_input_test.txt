Enter Username: 
Enter User Name To Add Credit To: 
Enter Amount To Add: 
